src/App.tsx

```diff
@@ -1,26 +1,167 @@
-/* eslint-disable max-len */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import {
+  USER_ID,
+  getTodos,
+  createTodo,
+  deleteTodo,
+  updateToDo,
+} from './api/todos'; // Ensure createTodo is imported
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { Header, ToDoList, Error, Footer } from './components';
 
-const USER_ID = 0;
+function getVisibleToDos(newTodos: Todo[], newStatus: Status) {
+  switch (newStatus) {
+    case Status.Active:
+      return newTodos.filter(todo => !todo.completed);
+
+    case Status.Completed:
+      return newTodos.filter(todo => todo.completed);
+
+    default:
+      return newTodos;
+  }
+}
```

Comment:
You can move it to a separete file. 
Then use it as util/helper in this component.